Src/CZICmd/cmdlineoptions.cpp

```diff
@@ -453,9 +453,9 @@ class CustomFormatter : public CLI::Formatter
 
         const auto result_from_stock_implementation = this->CLI::Formatter::make_usage(app, name);
         ostringstream ss(result_from_stock_implementation);
-        int majorVersion, minorVersion;
-        libCZI::GetLibCZIVersion(&majorVersion, &minorVersion);
-        ss << result_from_stock_implementation << endl << "  using libCZI version " << majorVersion << "." << minorVersion << endl;
+        int majorVersion, minorVersion, patchVersion;
+        libCZI::GetLibCZIVersion(&majorVersion, &minorVersion, &patchVersion);
+        ss << result_from_stock_implementation << endl << "  using libCZI version " << majorVersion << "." << minorVersion << "." << patchVersion << endl;
```

Comment:
Not sure why, but for some reason I locally get this after building this branch:

![image](https://user-images.githubusercontent.com/9250590/215991107-90f299dd-89f7-4b99-b9de-68ea4cb03d06.png)

Instead of 0.47.0, I am getting 0.39.0 ...

Reply:
Very good catch! ...and I have an idea why this is, actually a subtle one.

Reply:
fixed - problem was... when CMake is generating the config-file, it fills in the values of the version numbers ([here](https://github.com/ZEISS/libczi/blob/08c9b0fc4ba618f78151cccfd4c3640059cb6ead/Src/libCZI/libCZI_Config.h.in#L7)). But - we only have numbers filled in for the parts we give in the project-statement. Trouble now was - there is no number for "tweak", and tweak was replaced as "empty string", and therefore could not be interpreted as a number. That's why I had the idea of "let's prepend a '0') here, then an empty string would give '0' and done. What I missed was - a leading zero means "octal encoding", so the number was interpreted as an octal number.
It is now changed to "pass the version numbers" as a string.
![image](https://user-images.githubusercontent.com/4881321/216026909-389a6507-b0af-4b7b-b4c3-b4d1158562d3.png)
Thanks again!

Reply:
Works now as expected.